api/controllers/InvitationController.js

```diff
@@ -125,5 +125,26 @@ module.exports = {
         })
       })
     .then(results => res.ok({results}))
+  },
+
+  reInviteAll: function (req, res) {
+    const communityId = res.locals.community.id
+    return Invitation.where({community_id: communityId, used_by_id: null})
+    .fetchAll({withRelated: ['creator', 'community', 'tag']})
+    .then(invitations =>
+      Promise.map(invitations.models, invitation => {
```

Comment:
this should be queued because it could take quite a long time if there are lots of unsent invitations